launcher/ui/widgets/ModListView.cpp

```diff
@@ -65,19 +62,6 @@ void ModListView::setModel ( QAbstractItemModel* model )
         for(int i = 1; i < head->count(); i++)
             head->setSectionResizeMode(i, QHeaderView::ResizeToContents);
     }
-
-    auto real_model = model;
-    if (auto proxy_model = dynamic_cast<QSortFilterProxyModel*>(model); proxy_model)
-        real_model = proxy_model->sourceModel();
-
-    if (auto mod_model = dynamic_cast<ModFolderModel*>(real_model); mod_model) {
-        connect(mod_model, &ModFolderModel::updateFinished, this, [this, mod_model]{
-            auto mods = mod_model->allMods();
-            // Hide the 'Provider' column if no mod has a defined provider!
-            setColumnHidden(ModFolderModel::Columns::ProviderColumn,
-                    std::none_of(mods.constBegin(), mods.constEnd(), [](auto const mod){ return mod->provider().has_value(); }));
-        });
-    }
```

Comment:
whoa! why are we removing this?

Reply:
Because it means the user's choice whether the provider column is hidden/shown is ignored

Reply:
ah, right. the image column pr was merged so it's a proper setting now.